README.md

```diff
@@ -281,6 +281,38 @@ const myStorage = {
 const myStore = createStore(myStorage)
 ```
 
+##Handling errors
+### Errors in multi-stores
+Multi-stores have built-in error handling methods too. 
+You can customize the behavior by passing an object as the second parameter when creating your multi-store.
+
+```js
+import localStorageAdapter from 'browserstore.js/es/adapters/localStorage'
+import sessionStorageAdapter from 'browserstore.js/es/adapters/sessionStorage'
+import { createStore, multiStore } from 'browserstore.js'
+
+const stores = multiStore([
+  createStore(localStorageAdapter),
+  createStore(sessionStorageAdapter)
+], {
+  onGetError(err, key) {
+    return err.nextStore.get(key)
+  },
+  onSetError(err, key) {
+    err.currentStore.clear()
+    return err.nextStore.set(key)
+  },
+  onClearError(err) {
+    // Do nothing and ignore the error
+  },
+  onRemoveError(err, key) {
+    // Do nothing and ignore the error
+  },
+})
+```
```

Comment:
This takes into account the suggested change to the API.

```suggestion
```js
import localStorageAdapter from 'browserstore.js/es/adapters/localStorage'
import sessionStorageAdapter from 'browserstore.js/es/adapters/sessionStorage'
import { createStore, multiStore } from 'browserstore.js'
const stores = multiStore(
  [createStore(localStorageAdapter), createStore(sessionStorageAdapter)],
  {
    onGetError(err, key, currentStore, nextStore) {
      return nextStore.get(key)
    },
    onSetError(err, key, currentStore, nextStore) {
      currentStore.clear()
      return currentStore.set(key)
    },
    onClearError(err) {
      console.error(err)
    },
    onRemoveError(err, key) {
      console.error(err, key)
    }
  }
)
```
```